### DISCUSSION TITLE
谷歌云盘使用alist-proxy代理后，下载401错误

### DISCUSSION THREAD

**Discussion by xszrh** [AlistGo/alist#2321] [Q&A]
版本是V3
alist挂载谷歌云盘后，在另外一台VPS上开启alist-proxy代理。
已经做的设置：
1.config.json已填入site_url，格式为https://XXX.XXX
2.已在后台其他设置中找到token：alist-d1eXXXXXXXXXXXXXAOQgXOtqernpkY3
3.使用alist-proxy开启代理，未开启https，命令格式：
> root@XXXXc:~# ./alist-proxy -address https://XXX.XXX -token alist-d1eXXXXXXXXXXXXXAOQgXOtqernpkY3
listen and serve: :5243

显示alist-proxy已经开始工作，VPS端口已打开
4.在alsit后台填入谷歌云盘的下载代理URL，格式为http://11.11.11.11:5243
结果：
在alist前台下载谷歌云盘中的文件，只能下载到一个33字节的文件，如图
![1](https://user-images.githubusercontent.com/117139845/201513636-5ab63f24-f513-430a-8488-e2e408d10368.png)

使用记事本打开，内容为{"code":401,"msg":"sign invalid"}，如图
![2](https://user-images.githubusercontent.com/117139845/201513713-f86d5540-887b-4356-b054-73bfb66f93f1.png)

如果下载的是一个txt文件，页面直接显示{"code":401,"msg":"sign invalid"}，如图
![3](https://user-images.githubusercontent.com/117139845/201513769-d18c73da-5f13-4886-a991-a6e2241b72b6.png)

自己感觉应该是sign验证出现了问题，是alist-proxy参数输入的有问题，还是哪里的设置出现了问题，自己已经尝试过很多次了，一直出现这种错误。有遇到同样情况的吗？请大佬看看能不能帮忙提示一下，非常感谢。

**Comment by Minerchu** [comment]
如果token和地址都没错的话，建议你把alist更新一下。遇到过类似问题，老版本不支持，虽然也是3.0。

**Comment by zi4dbek** [comment]
更新alist-proxy版本就好了，我也是这么好的
